source/mainnet/net/guides/wallet-sdk/wallet-sdk-identity-creation.rst

```diff
@@ -83,7 +83,61 @@ The first step is to create the actual identity request. To do this, you need th
 
         Kotlin (Android)
 
-        TODO Write the Kotlin example.
+        .. code-block:: Kotlin
+
+            import cash.z.ecc.android.bip39.Mnemonics
+            import cash.z.ecc.android.bip39.toSeed
+            import com.concordium.example.wallet.services.ConcordiumWalletProxyService
+            import com.concordium.sdk.ClientV2
+            import com.concordium.sdk.Connection
+            import com.concordium.sdk.TLSConfig
+            import com.concordium.sdk.crypto.wallet.ConcordiumHdWallet
+            import com.concordium.sdk.crypto.wallet.Identity
+            import com.concordium.sdk.crypto.wallet.IdentityRequestInput
+            import com.concordium.sdk.crypto.wallet.Network
+            import com.concordium.sdk.requests.BlockQuery
+
+            fun createIdentityRequest(): String {
+                // Select the identity provider that is to be used for creating the identity. Here we
+                // choose the first one in the list available from the Concordium wallet-proxy.
+                val identityProvider = getIdentityProviders(walletProxyTestnetBaseUrl)[0]
+
+                // The index of the identity to create. The index is incremented per identity created
+                // for a specific identity provider. For the first identity created it should be 0,
+                // for the second it should be 1, and so forth.
+                val identityIndex = 0
+
+                val connection = Connection.newBuilder()
+                    .host(nodeAddress)
+                    .port(nodePort)
+                    .useTLS(TLSConfig.auto())
+                    .build()
+                val client = ClientV2.from(connection)
+                val cryptographicParameters = client.getCryptographicParameters(BlockQuery.BEST)
+
+                val seedPhrase = "fence tongue sell large master side flock bronze ice accident what humble bring heart swear record valley party jar caution horn cushion endorse position"
+                @OptIn(ExperimentalStdlibApi::class)
+                val seedAsHex = Mnemonics.MnemonicCode(seedPhrase.toCharArray()).toSeed().toHexString()
+                val wallet = ConcordiumHdWallet.fromHex(seedAsHex, Network.TESTNET)
```

Comment:
```suggestion
                val wallet = ConcordiumHdWallet.fromHex(seedAsHex, Network.TESTNET)  // Or Network.MAINNET, if working on mainnet.
```
We also have it on the transaction page, but I think it's most important to have on the first page.